shared/src/constants/constants.ts

```diff
@@ -90,6 +90,7 @@ export const RetailVaultList = [
   "rstETH-THETA",
   "rBTC-THETA",
   "rAVAX-THETA",
+  "rUSDC-AVAX-P-THETA",
```

Comment:
Need to add this into the `ProdExcludeVault` for now

Reply:
Done.